string frame_id
int8 x
int8 y
int8 battery
int8 work
string display_action
int32 time_away_from_network
string current_state

mines/task[] task_list
mines/region[] region_list
mines/region_state[] region_state_list
